flutter/lib/src/integrations/native_app_start_integration.dart

```diff
@@ -13,6 +15,36 @@ class NativeAppStartIntegration extends Integration<SentryFlutterOptions> {
   final SentryNative _native;
   final SchedulerBindingProvider _schedulerBindingProvider;
```

Comment:
you can replace this with an instance of `FrameCallbackHandler`, as it will call the `SchedulerBinding` only when needed